scripts/setup.sh

```
@@ -256,6 +256,13 @@ wait_for_namespace_cleanup "${INSTALLER_NAMESPACE}"
 # Apply kustomize overlay
 oc apply -k overlays/${INSTALLER_KUSTOMIZE_OVERLAY}
 
+# Create fulfillment-controller OAuth credentials (client registered in Keycloak realm)
+oc create secret generic fulfillment-controller-credentials \
+    --from-literal=client-id=fulfillment-controller \
+    --from-literal=client-secret=t2N7qNiOYZzt6InsHOVsV5AzcZZbA7Sg \
+    -n ${INSTALLER_NAMESPACE} \
```

Shellcheck (0.11.0)

[info] Double quote to prevent globbing and word splitting.

(SC2086)